### DISCUSSION TITLE
Before Converting " Publickeys_in.txt to Xpoints Using a Python Tool Should i Put All the List of Pubkeys with or without (03) Prefix ?

### DISCUSSION THREAD

**Discussion by dragonhlf8** [community/community#60259] [Programming Help]
### Select Topic Area

Question

### Body

Before Converting to Xpoints in " Publickey_in.txt Should i Put All the List of Pubkeys like this -
without (03) Prefix ( a2efa402fd5268400c77c20e574ba86409ededee7c4020e4b9f0edbee53de0d4 ) or Like this ' 03a2efa402fd5268400c77c20e574ba86409ededee7c4020e4b9f0edbee53de0d4 )

**Comment by kanghthai** [comment]
Before converting

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @dragonhlf8 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
